odbc-api/tests/integration.rs

```diff
@@ -2767,6 +2773,7 @@ fn no_data(profile: &Profile) {
 }
 
 /// List tables for various data sources
+/// Table name comparison is insensitive on Windows
```

Comment:
I do not understand this comment. Test runs fine on my local win64 without converting to lowercase.

If the comment refers to the search patterns for table or columns: Wether its case is significant should be determined by the `SQL_ATTR_METADATA` attribute. See: <https://docs.microsoft.com/en-us/sql/odbc/reference/syntax/sqltables-function?view=sql-server-ver15>

Reply:
> Test runs fine on my local win64
Do you setup the database through Docker over Linux? Or does it run as a native Windows program

> If the comment refers to the search patterns for table or columns
The search patterns are ok. The problem is, when you run this the table name returned is all lower case
Ej: Asks for "MyTableName", gets "mytablename"

There are many databases who do this by default (Postgres for example), unless the table name has been explicitly indicated to be sensitive on creation

I'm gonna adjust the test to check for both cases where it makes sense

Reply:
Thanks for clarifying! I do run these through docker over Linux. It's fine to leave it as is, I just did not understand at firs.